locales/en-US/app.ftl

```diff
@@ -146,7 +148,7 @@ tourLink = Launch Tour
 contribute = Contribute
 bugReports = Bug Reports
 discussExperiment = Discuss { $title }
-tourOnboardingTitle = {$title} enabled!
+tourOnboardingTitleGeneral = {$title}
```

Comment:
Since the string is just the experiment name, do you need this string at all exposed for localization?

Reply:
good point, I can just remove this, thanks